packages/layouts-css/src/page.scss

```diff
@@ -46,22 +46,97 @@
+
       padding: var(--iui-size-xl) unquote('max(var(--iui-size-2xl), env(safe-area-inset-right))') var(--iui-size-xl) unquote('max(var(--iui-size-2xl), env(safe-area-inset-left))');
 
       @include small-monitor {
-        padding: var(--iui-size-l) unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))') var(--iui-size-l) unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
+        padding: var(--iui-size-l)
+          unquote(
+            'max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))'
+          )
+          var(--iui-size-l)
+          unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
       }
 
       @include tablet {
-        padding: var(--iui-size-m) unquote('max(var(--iui-size-xl), env(safe-area-inset-right))') var(--iui-size-m) unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
+        padding: var(--iui-size-m)
+          unquote('max(var(--iui-size-xl), env(safe-area-inset-right))')
+          var(--iui-size-m)
+          unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
       }
 
       @include mobile {
-        padding: var(--iui-size-xs) unquote('max(var(--iui-size-l), env(safe-area-inset-right))') var(--iui-size-xs) unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
+        padding: var(--iui-size-xs)
+          unquote('max(var(--iui-size-l), env(safe-area-inset-right))')
+          var(--iui-size-xs)
+          unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
       }
       //#endregion Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
     }
 
+
+    &-tools {
+      display: flex;
+      flex-direction: row;
+      margin-bottom: calc(var(--iui-size-s) * 2);
+      justify-content: space-between;
+
+      &-left,
+      &-right {
+        display: flex;
+        justify-content: flex-start;
+        flex-direction: row;
+        gap: var(--iui-size-xs);
+        align-items: center;
+        min-width: 0;
+        flex-basis: 50%;
+      }
+
+      &-right {
+        justify-content: flex-end;
+      }
+    }
+
+
+    //#region Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
+
+    @include small-monitor {
+      padding-inline: unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-right))') unquote('max(calc(var(--iui-size-l) * 2), env(safe-area-inset-left))');
+    }
+
+    @include tablet {
+      padding-inline: unquote('max(var(--iui-size-xl), env(safe-area-inset-right))') unquote('max(var(--iui-size-xl), env(safe-area-inset-left))');
+    }
+
+    @include mobile {
+      padding: unquote('max(var(--iui-size-l), env(safe-area-inset-right))') unquote('max(var(--iui-size-l), env(safe-area-inset-left))');
+    }
+    //#endregion Add env(safe-area-inset-*) for iPhone notch on the left & right sides of content area
+
+    &-zones {
```

Comment:
What "-zones" are for?

Reply:
Oops that is leftover styling of previous layout. Removed.